GentWebApi/Controllers/EventsController.cs

```diff
@@ -19,6 +17,11 @@ public EventsController(GentDbContext context)
 			_context = context;
 		}
 
+		[HttpGet]
+		public ActionResult<IEnumerable<Event>> Get() {
+			return _context.Events;
+		}
+
```

Comment:
Alle events ophalen is mss een beetje te veel, mss nog een check op zetten dat einddatum binnen de komende twee maanden is

Reply:
das oke hoor het duurt maar 300ms om alles op te halen, en de subscribed branches ga ik dan sorteren op datum en misschien dan enkel uit de laatste zoveel maand ophalen

Reply:
ja maar als ge 1000 entrepreneurs hebt die elk hun events hebben :p. En ge kunt hier in de backend al filteren zoda ge enkel die van de komende twee maanden teruggeeft eh

Reply:
oke fair enough kzalt int lijstje zetten